docs/LOCAL_DEPLOYMENT.md

```diff
@@ -7,6 +7,9 @@ This document and other documents, referenced here, contain steps to git clone t
 ### Steps to clone the restructured repository
 #### If you have an existing cloned repository
 1) Use recursive clone to clone the repository and the sub-modules.
+```
+git clone --recurse-submodules  https://github.com/Azure/azure-iot-pcs-remote-monitoring-java.git
+```
```

Comment:
The link [here](https://github.com/Azure/azure-iot-pcs-remote-monitoring-java/pull/62/files#diff-68bcf8b9f8ec9df423480fc978e76664R30) is not valid.